code/game/gamemodes/game_mode.dm

```diff
@@ -533,3 +533,14 @@
 		return 0
 
 	return max(0, enemy_minimum_age - C.player_age)
+
+
+/datum/game_mode/proc/replace_jobbaned_player(mob/living/M, role_type)
+	var/list/mob/dead/observer/candidates = pollCandidates("Do you want to play as a [role_type]?", "[role_type]", null, FALSE, 100)

```

Comment:
Use the special flag argument for pollCandidates so people who have that team antag turned off don't get the popup.